### PR TITLE
Updated to allow configuration of Giphy UI

### PR DESCRIPTION
**Pull request by fred-bowker**
Hi, I created a new Pull request to remove the isKeyboardShown change

### CODE REVIEW

File: Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift [hunk 1]
```diff
@@ -190,10 +189,10 @@ public struct MessageField: View {
...
         .sheet(isPresented: $isGIFPickerPresented) {
             if let giphyKey = configuration.giphyKey {
-                GiphyPicker(giphyKey: giphyKey)
-                    .ignoresSafeArea()
-                    .presentationDetents([.fraction(0.9)])
-                    .presentationDragIndicator(.hidden)
+                GiphyPicker(giphyKey: giphyKey, giphyConfig: configuration.giphyConfig)
+                        .ignoresSafeArea()
+                        .presentationDetents([.fraction(configuration.giphyConfig.presentationDetents)])
+                        .presentationDragIndicator(.hidden)
```
Comment (jaesung-0o0) [inline Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift:195]:
```suggestion
                GiphyPicker(
                    giphyKey: giphyKey, 
                    giphyConfig: configuration.giphyConfig
                )
                .ignoresSafeArea()
                .presentationDetents(
                    [.fraction(configuration.giphyConfig.presentationDetents)]
                )
                .presentationDragIndicator(.hidden)
```

Comment (jaesung-0o0) [reply Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift:195]: I left the minor suggestion about code convention.

**Review by jaesung-0o0 — APPROVED** [review]
@fred-bowker 
Sorry for the late reviews.
This is a really nice request. Thanks for your PR. 🙌
I left a minor suggestion, but it's optional.
In the future, It would be good to move `giphyKey` in `ChatConfiguration` to `GiphyConfiguration`

LGTM👍

File: Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift (hunk 1 above)
Comment (fred-bowker) [reply Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift:195]: cool can we merge this

Comment (jaesung-0o0) [reply Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift:195]: Of course!

**Pull request merged**